### PR TITLE
Remove hard-coded namespace from k8s config

### PR DESCRIPTION
**Pull request by jpeeler**
See commit text for more details. I haven't actually tested this yet, but looking at the client code I can't see how this would fail in any previously working scenario. @tarasglek care to verify all is well now?

(#5378)

### CODE REVIEW

File: libbeat/processors/add_kubernetes_metadata/config.go [hunk 1]
```diff
@@ -30,12 +30,11 @@ type Enabled struct {
 
 type PluginConfig []map[string]common.Config
 
-func defaultKuberentesAnnotatorConfig() kubeAnnotatorConfig {
+func defaultKubernetesAnnotatorConfig() kubeAnnotatorConfig {
 	return kubeAnnotatorConfig{
 		InCluster:       true,
 		SyncPeriod:      1 * time.Second,
 		CleanupTimeout:  60 * time.Second,
-		Namespace:       "kube-system",
```
Comment (exekias) [inline libbeat/processors/add_kubernetes_metadata/config.go]: I think it's best if we remove `Namespace` field from the kubeAnnotatorConfig, and move all references to `client.Namespace`

**Comment by exekias** [comment]
jenkins, test it please

File: libbeat/processors/add_kubernetes_metadata/config.go (hunk 1 above)
Comment (jpeeler) [reply libbeat/processors/add_kubernetes_metadata/config.go]: Yeah, I considered doing that initially as well. Will adjust now!

**Comment by exekias** [comment]
Just gave this a try, works for me, thank you :tada:

**Pull request merged**